doc/api/events.md

```diff
@@ -663,9 +663,13 @@ added: v11.13.0
 
 Creates a `Promise` that is resolved when the `EventEmitter` emits the given
 event or that is rejected when the `EventEmitter` emits `'error'`.
-The `Promise` will resolve with an array of all the arguments emitted to the
+The `Promise` will fulfill with an array of all the arguments emitted to the
 given event.
 
+Note: This method is intentionally generic and works with web platform
+[EventTarget](WHATWG-EventTarget) instances. `EventTarget`s have no special
```

Comment:
```suggestion
[EventTarget](WHATWG-EventTarget) interface, which have no special
```

Argubly, there are not actual `EventTarget` instances in Node.js, just look-alike objects with a similar (subset?) of the interface.